components/style/properties/gecko.mako.rs

```diff
@@ -1338,17 +1361,7 @@ ${impl_style_struct(style_struct)}
 ${define_ffi_struct_accessor(style_struct)}
 % endfor
 
-lazy_static! {
-    pub static ref INITIAL_GECKO_VALUES: ComputedValues = ComputedValues {
-        % for style_struct in data.style_structs:
-           ${style_struct.ident}: style_structs::${style_struct.name}::initial(),
-        % endfor
-        custom_properties: None,
-        shareable: true,
-        writing_mode: WritingMode::empty(),
-        root_font_size: longhands::font_size::get_initial_value(),
-    };
-}
+static mut INITIAL_GECKO_VALUES: *mut ComputedValues = 0 as *mut ComputedValues;
 
```

Comment:
@eddyb @Manishearth Can you describe more concretely what the problem is here?

Basically, the semantics I want are:
(1) On the main thread, at some point early in startup where we know there are no other threads, we want to initialize the thing.
(2) The thing is mutable during the interesting / multithreaded parts of the lifetime of the program.
(3) Once all those threads have been shut down, we want to deallocate the thing.

I'm definitely coming at this from a pretty C++-y perspective, so maybe there's a better way of doing it. Here are my constraints:
* I want to be able to explicitly destroy the thing before the program terminates, so that Gecko's leak checking can note the destructor calls and avoid reporting spurious leaks.
* I don't want to incur noticeable synchronization overhead when using these things during the interesting parts of the program, because they're totally immutable at that point.

How would you suggest achieve these?